### PR TITLE
Rewrite Contributing and readme

### LINKED ISSUE #64: Level up README and related markdown content
Level up and cleanup markdown content. 

See [.github#8](https://github.com/opensearch-project/.github/issues/8) for details and recommendations, and [common-utils](https://github.com/opensearch-project/common-utils/) for an example.

- [x] README.md
- [x] CONTRIBUTING.md
- [ ] DEVELOPER_GUIDE.md
- [x] MAINTAINERS.md
- [ ] ADMINS.md
- [ ] SECURITY.md
- [ ] RELEASING.md
- [ ] TESTING.md
- [ ] other .md files

### PR DESCRIPTION
**Pull request by kolchfa-aws**
### Description
Rewrite Contributing and readme

### Issues Resolved
Fixes #64

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,60 +1,123 @@
...
-There are two ways to contribute: 1) create an issue where you describe the change and 2) create a pull request that contains the content to directly insert into the documentation.
+There are two ways to contribute: 
+1. [Create an issue](#create-an-issue) asking us to change the documentation.
+2. [Contribute documentation content](#contribute-content) yourself.
 
-**Create an Issue**
+## Create an issue
 
-Use the Github issue tracker to describe the change you'd like to make. 
+Use the GitHub issue tracker to describe the change you'd like to make:
```
Comment (hdhalter) [inline CONTRIBUTING.md:17]:
```suggestion
Use the documentation issue template to describe the change you'd like to make.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,60 +1,123 @@
...
-Before entering a change request, please read this document carefully. We want to make sure we have all the information necessary to effectively respond to your contribution. 
+## Contributing to the documentation site
 
-There are two ways to contribute: 1) create an issue where you describe the change and 2) create a pull request that contains the content to directly insert into the documentation.
+There are two ways to contribute: 
+1. [Create an issue](#create-an-issue) asking us to change the documentation.
+2. [Contribute documentation content](#contribute-content) yourself.
 
-**Create an Issue**
+## Create an issue
```
Comment (hdhalter) [inline CONTRIBUTING.md:15]:
```suggestion
### Create an issue
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,60 +1,123 @@
...
 
 1. Go to https://github.com/opensearch-project/documentation-website/issues and select *New issue*.
-2. Enter the requested information and include as much detail as possible, especially which version or versions the request affects.
+2. Enter the requested information, including as much detail as possible, especially which version or versions the request affects.
 3. Select *Submit new issue*. 
 
+The `untriaged` label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
 
-The ‘untriaged’ label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+## Contribute content
```
Comment (hdhalter) [inline CONTRIBUTING.md:25]:
```suggestion
### Contribute content
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,60 +1,123 @@
...
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE)
+- [API Style Guide](API_STYLE_GUIDE)
+- [Formatting Guide](FORMATTING_GUIDE) 
 
-1. Fork the repository.
-2. Modify the source. Make sure to focus on the specific change you are contributing. For example, if you also reformat the code, it will be hard for us to focus on your change.
-3. Test the code. Make sure that any local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Create the [pull request](https://github.com/opensearch-project/documentation-website/pulls) and save it, making sure to answer the default questions in the pull request template.
+### Editing files in GitHub
```
Comment (hdhalter) [inline CONTRIBUTING.md:49]: I'm confused about these headings. I'll reach out offline.

**Review by hdhalter — COMMENTED** [review]
Thanks, Fanit! I have some questions about the flow of info on the contributions page.

File: CONTRIBUTING.md (hunk 4 above)
Comment (kolchfa-aws) [reply CONTRIBUTING.md]: @hdhalter I made some changes to the structure of the doc. Please review again. Thanks.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,60 +1,139 @@
...
-Before entering a change request, please read this document carefully. We want to make sure we have all the information necessary to effectively respond to your contribution. 
+1. Go to https://github.com/opensearch-project/documentation-website/issues and select **New issue**.
+1. Enter the requested information, including as much detail as possible, especially which version or versions the request affects.
+1. Select **Submit new issue**. 
 
-There are two ways to contribute: 1) create an issue where you describe the change and 2) create a pull request that contains the content to directly insert into the documentation.
+The `untriaged` label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
 
-**Create an Issue**
+## Finding contributions to work on
```
Comment (hdhalter) [inline CONTRIBUTING.md:29]: Can we add this as a note underneath "contributing content"?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,60 +1,139 @@
...
-3. Select *Submit new issue*. 
+## Contributing content
 
+There are two ways to contribute content depending on the magnitude of the change:
 
-The ‘untriaged’ label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+- [Minor changes](#making-minor-changes): For small changes like fixing a typo or adding a parameter, you can edit files in GitHub directly. This approach does not require cloning the repository and does not let you test the documentation.
+- [Major changes](#making-major-changes): For changes you want to test first, like reorganizing pages or adding a table or section, you can edit files locally and push the changes to GitHub. This approach requires setting up a local version of the repository and lets you test documentation.
 
+### Documentation workflow
```
Comment (hdhalter) [inline CONTRIBUTING.md:40]:
```suggestion
### Contribution workflow
```
?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,60 +1,139 @@
...
 
-**Create a pull request**
+The workflow for contributing documentation is no different than the one for contributing code:
 
-If you’d like to make a change directly to the content, create a pull request. If it’s a quick fix, we should be able to release the update quickly. Bigger requests might take a bit of time for us to review. 
+- Make your changes
+- Build locally to check your work (only possible if you are making changes locally)
```
Comment (hdhalter) [inline CONTRIBUTING.md:45]:
```suggestion
- Build a local instance of the content to check your work (only possible if you are editing the files locally)
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,60 +1,139 @@
...
+- Submit a [pull request](https://github.com/opensearch-project/documentation-website/pulls) (PR)
+- Maintainers review and merge your PR
 
-Note that a pull request requires DCO sign off before we can merge it. You can use the -s command line option to append this automatically to your commit message, for example $ git commit -s -m 'This is my commit message'. For more information, see https://github.com/apps/dco.
+If the change requires significant work, open an issue where we can first discuss your request.
```
Comment (hdhalter) [inline CONTRIBUTING.md:49]: Not sure if this is needed since we already included reorganizing pages in a previous section.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,60 +1,139 @@
...
-* Check existing open and recently merged pull requests to ensure that someone else hasn't addressed the problem already.
-* If the change requires significant work, open an issue where we can first discuss your request.
+Before contributing content, make sure to read the following resources:
+- [README](README.md)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md)
+- [API Style Guide](API_STYLE_GUIDE.md)
+- [Formatting Guide](FORMATTING_GUIDE.md) 
 
-To create a pull request:
+NOTE: Please make sure that any documentation you submit is your work or work you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution.
```
Comment (hdhalter) [inline CONTRIBUTING.md:59]:
```suggestion
NOTE: Please make sure that any documentation you submit is your work or work that you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution. 
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,60 +1,139 @@
...
 
-1. Fork the repository.
-2. Modify the source. Make sure to focus on the specific change you are contributing. For example, if you also reformat the code, it will be hard for us to focus on your change.
-3. Test the code. Make sure that any local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Create the [pull request](https://github.com/opensearch-project/documentation-website/pulls) and save it, making sure to answer the default questions in the pull request template.
+### Making minor changes
 
-If you need additional support, GitHub provides documentation on [forking a repository](https://help.github.com/articles/fork-a-repo/) and [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+If you want to add a few paragraphs to a file, try this approach:
```
Comment (hdhalter) [inline CONTRIBUTING.md:63]:
```suggestion
If you want to make minor changes to an existing file, you can use this approach:
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,60 +1,139 @@
...
-## Finding contributions to work on
+1. In your fork on GitHub, navigate to the file that you want to change.
+
+1. In the upper-right corner, select the dropdown with the pencil icon and select **Edit in place**. Edit the file.
+
+1. In the upper-right corner, select **Commit changes...***. Enter the commit message and optional description and select **Create a new branch for this commit and start a pull request**.
+
+### Making major changes
+
+If you're making major changes to the documentation and need to see the rendered HTML before submitting a pull request, you need to work in a local copy of the repository.
```
Comment (hdhalter) [inline CONTRIBUTING.md:75]:
```suggestion
If you're adding new pages or making major changes to the documentation, we recommend that you work in a local copy of the repository and test the rendered HTML before submitting a pull request. 
```

**Review by hdhalter — COMMENTED** [review]
A few comments. Thanks!

File: CONTRIBUTING.md (hunk 7 above)
Comment (kolchfa-aws) [reply CONTRIBUTING.md:45]: Reworded.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -1,60 +1,134 @@
...
+1. Enter the requested information, including as much detail as possible, especially which version or versions the request affects.
+1. Select **Submit new issue**. 
+
+The `untriaged` label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+
+## Contributing content
+
+There are two ways to contribute content depending on the magnitude of the change:
+
+- [Minor changes](#making-minor-changes): For small changes like fixing a typo or adding a parameter, you can edit files in GitHub directly. This approach does not require cloning the repository and does not let you test the documentation.
```
Comment (hdhalter) [inline CONTRIBUTING.md:34]:
```suggestion
- [Minor changes](#making-minor-changes): For small changes to existing files, like fixing typos or adding parameters, you can edit files in GitHub directly. This approach does not require cloning the repository and does not let you test the documentation.
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -1,60 +1,134 @@
...
+- [Major changes](#making-major-changes): For changes you want to test first, like reorganizing pages or adding a table or section, you can edit files locally and push the changes to GitHub. This approach requires setting up a local version of the repository and lets you test documentation.
```
Comment (hdhalter) [inline CONTRIBUTING.md:35]:
```suggestion
- [Major changes](#making-major-changes): For changes you want to test first, like adding new or reorganizing pages or adding a table or section, you can edit files locally and push the changes to GitHub. This approach requires setting up a local version of the repository and lets you test documentation.
```

**Review by hdhalter — COMMENTED** [review]
Looks good, just a couple of minor suggestions.

File: README.md [hunk 14]
```diff
@@ -4,33 +4,9 @@
...
 
-This repository contains [Markdown](https://guides.github.com/features/mastering-markdown/) files organized into Jekyll "collections" (e.g., `_search-plugins`, `_opensearch`, etc.). Each Markdown file correlates with one page on the website.
-
-Using plain text on GitHub has many advantages:
-
-- Everything is free, open source, and works on every operating system. Use your favorite text editor, Ruby, Jekyll, and Git.
-- Markdown is easy to learn and looks good in side-by-side diffs.
-- The workflow is no different than contributing code. Make your changes, build locally to check your work, and submit a pull request. Reviewers check the PR before merging.
-- Alternatives like wikis and WordPress are full web applications that require databases and ongoing maintenance. They also have inferior versioning and content review processes compared to Git. Static websites, such as the ones Jekyll produces, are faster, more secure, and more stable.
+This repository contains [Markdown](https://guides.github.com/features/mastering-markdown/) files organized into Jekyll _collections_ (for example, `_api-reference` or `_dashboards`). Each Markdown file corresponds to one page on the website.
```
Comment (hdhalter) [inline README.md:9]: Do we need this section here? Would it be more appropriate in the contribution file?

Comment (kolchfa-aws) [reply README.md:9]: The readme is the what and the contributing is the how in my opinion. So readme is about this repo and contributing is about how to contribute. So I think it's better to have it here.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -1,60 +1,144 @@
...
+      - [Setting up your local copy of the repository](#setting-up-your-local-copy-of-the-repository)
+      - [Making, viewing, and submitting changes](#making-viewing-and-submitting-changes)
+- [Review process](#review-process)
+   - [Style linting](#style-linting)
+- [Getting help](#getting-help)
+
 # Contributing Guidelines
 
-Thank you for your interest in improving the OpenSource documentation! We value and appreciate all feedback and contributions from our community, including requests for additional documentation, corrections to existing content, and to report technical issues with the documentation site. 
+Thank you for your interest in improving the OpenSource documentation! We value and appreciate all feedback and contributions from our community, including requests for additional documentation, corrections to existing content, and reports of technical issues with the documentation site.
```
Comment (hdhalter) [inline CONTRIBUTING.md:15]:
```suggestion
Thank you for your interest in improving the OpenSearch documentation! We value and appreciate all feedback and contributions from our community, including requests for additional documentation, corrections to existing content, and reports of technical issues with the documentation site. 
```

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -1,60 +1,144 @@
...
+   - [Before you start](#before-you-start)
+   - [Making minor changes](#making-minor-changes)
+   - [Making major changes](#making-major-changes)
+      - [Setting up your local copy of the repository](#setting-up-your-local-copy-of-the-repository)
+      - [Making, viewing, and submitting changes](#making-viewing-and-submitting-changes)
+- [Review process](#review-process)
+   - [Style linting](#style-linting)
+- [Getting help](#getting-help)
+
 # Contributing Guidelines
```
Comment (hdhalter) [inline CONTRIBUTING.md:13]:
```suggestion
# Contributing guidelines
```

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -1,60 +1,144 @@
+- [Creating an issue](#creating-an-issue)
+- [Contributing content](#contributing-content)
+   - [Documentation workflow](#documentation-workflow)
```
Comment (hdhalter) [inline CONTRIBUTING.md:3]:
```suggestion
   - [Contribution workflow](#contribution-workflow)
```

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -1,60 +1,144 @@
...
+- Build the documentation website to check your work (only possible if you are making changes locally)
+- Submit a [pull request](https://github.com/opensearch-project/documentation-website/pulls) (PR)
+- Maintainers review and merge your PR
+
+### Before you start
+
+Before contributing content, make sure to read the following resources:
+- [README](README.md)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md)
+- [API Style Guide](API_STYLE_GUIDE.md)
```
Comment (hdhalter) [inline CONTRIBUTING.md:52]: - [Terms](TERMS.ms)

**Review by hdhalter — APPROVED** [review]
Just a few tweaks, but otherwise, looks good!

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -1,60 +1,144 @@
...
+Thank you for your interest in improving the OpenSearch documentation! We value and appreciate all feedback and contributions from our community, including requests for additional documentation, corrections to existing content, and reports of technical issues with the documentation site. 
 
-## How to request changes
+You can [create an issue](#creating-an-issue) asking us to change the documentation or [contribute content](#contributing-content) yourself.
 
-Before entering a change request, please read this document carefully. We want to make sure we have all the information necessary to effectively respond to your contribution. 
+NOTE: If you’d like to contribute but don't know where to start, try browsing existing [issues](https://github.com/opensearch-project/documentation-website/issues). Our projects use custom GitHub issue labels for status, version, type of request, and so on, but we recommend looking at any issues labeled `good first issue` first.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:19]: Should we mention the `help wanted` label as well?

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -1,60 +1,144 @@
...
-* Check existing open and recently merged pull requests to ensure that someone else hasn't addressed the problem already.
-* If the change requires significant work, open an issue where we can first discuss your request.
+Before contributing content, make sure to read the following resources:
+- [README](README.md)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md)
+- [API Style Guide](API_STYLE_GUIDE.md)
+- [Formatting Guide](FORMATTING_GUIDE.md) 
 
-To create a pull request:
+NOTE: Please make sure that any documentation you submit is your work or work that you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:55]:
```suggestion
NOTE: Make sure that any documentation you submit is your work or work that you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution. 
```

File: CONTRIBUTING.md [hunk 21]
```diff
@@ -1,60 +1,144 @@
...
-If you need additional support, GitHub provides documentation on [forking a repository](https://help.github.com/articles/fork-a-repo/) and [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+If you want to make minor changes to an existing file, you can use this approach:
 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-## Finding contributions to work on
+1. In your fork on GitHub, navigate to the file that you want to change.
 
-If you’d like to contribute but don't know where to start, try browsing existing issues. Our projects use custom GitHub issue labels for status, version, type of request, etc., but we recommend looking at any issues labeled “good first issue” first. 
+1. In the upper-right corner, select the dropdown with the pencil icon and select **Edit in place**. Edit the file.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:65]: Why do they have to use the "Edit in place" option? They can also just select the pencil icon, which defaults to editing through GitHub automatically?

File: CONTRIBUTING.md [hunk 22]
```diff
@@ -1,60 +1,144 @@
...
 
+1. In the upper-right corner, select **Commit changes...***. Enter the commit message and optional description and select **Create a new branch for this commit and start a pull request**.
 
-## Security issue notifications
+### Making major changes
 
-If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
+If you're adding new pages or making major changes to the documentation, we recommend that you work in a local copy of the repository and test the rendered HTML before submitting a pull request.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:71]: Besides new pages and sections, what else do we define as a major change? Maybe we can revise to:

```suggestion
If you're adding new pages or making major changes to the documentation, such as adding new images, pages, sections, or styling, we recommend that you work in a local copy of the repository and test the rendered HTML before submitting a pull request. 
```

File: CONTRIBUTING.md [hunk 23]
```diff
@@ -1,60 +1,144 @@
...
 
+#### Setting up your local copy of the repository
 
-## Licensing
+Follow these steps to set up your local copy of the repository:
 
-See the [LICENSE](LICENSE) file for our project's licensing. 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.
+
+1. Navigate to the repository root.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:79]:
```suggestion
1. Navigate to your cloned repository.
```

Comment (Naarcha-AWS) [reply CONTRIBUTING.md:79]: Technically speaking, they could navigate to anywhere in their repository clone and still make commits.

File: CONTRIBUTING.md [hunk 24]
```diff
@@ -1,60 +1,144 @@
...
+   ruby -v
+   ```
+
+1. Install [Jekyll](https://jekyllrb.com/) if you don't already have it:
+
+   ```
+   gem install bundler jekyll
+   ```
+
+1. Install dependencies:
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:95]:
```suggestion
1. Install the Jekyll dependencies:
```

File: CONTRIBUTING.md [hunk 25]
```diff
@@ -1,60 +1,144 @@
...
+   sh build.sh    
+   ```
+
+   The build script should automatically open your web browser, but if it doesn't, open [http://localhost:4000/docs/](http://localhost:4000/docs/).
+
+1. Create a new branch against the latest source on the main branch. 
+
+1. Edit the Markdown files that you want to change.
+
+1. When you save a file, marvel as Jekyll automatically rebuilds the site and refreshes your web browser. This process can take anywhere from 10 to 30 seconds.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:117]: The build takes longer on most machines now.

```suggestion
1. When you save a file, Jekyll automatically rebuilds the site and refreshes your web browser. This process can take anywhere from 60 - 90 seconds.
```

File: CONTRIBUTING.md [hunk 26]
```diff
@@ -1,60 +1,144 @@
...
+
+1. When you're happy with how everything looks, commit, [sign off](https://github.com/src-d/guide/blob/9171d013c648236c39faabcad8598be3c0cf8f56/developer-community/fix-DCO.md#how-to-prevent-missing-sign-offs-in-the-future), push your changes to your fork, and submit a pull request.
+
+    Note that a pull request requires DCO sign-off before we can merge it. You can use the -s command line option to append this automatically to your commit message, for example `git commit -s -m 'This is my commit message'`. For more information, see https://github.com/apps/dco.
+
+## Review process
+
+We greatly appreciate everyone who takes the time to make a contribution. We will review all contributions as quickly as possible. If it’s a quick fix, we should be able to release the update quickly. Bigger requests might take a bit of time for us to review. 
+
+During the PR process, expect that there will be some back-and-forth. Please try to respond to comments in a timely fashion, and if you don't want to continue with the PR, let us know.
```
Comment (Naarcha-AWS) [inline CONTRIBUTING.md:127]:
```suggestion
During the PR process, expect that there will be some back-and-forth. If you want to get your contribution merged quickly, try to respond to comments in a timely fashion, and if you don't want to continue with the PR, let us know. 
```

File: README.md [hunk 27]
```diff
@@ -21,141 +28,19 @@ If you encounter problems or have questions when contributing to the documentati
...
-
 ## Code of conduct
 
 This project has adopted an [Open Source Code of Conduct](https://opensearch.org/codeofconduct.html).
 
 
 ## Security
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public GitHub issue.
```
Comment (Naarcha-AWS) [inline README.md:38]:
```suggestion
If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Do **not** create a public GitHub issue.
```

File: README.md [hunk 28]
```diff
@@ -2,12 +2,19 @@
 
 # About the OpenSearch documentation repo
 
-The documentation repository contains the documentation for OpenSearch, the search, analytics, and visualization suite with advanced security, alerting, SQL support, automated index management, deep performance analysis, and more. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
+The documentation repository contains the documentation for OpenSearch, the highly scalable and extensible open-source software suite for search, analytics, observability, and other data-intensive applications. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
```
Comment (natebower) [inline README.md:5]: Should this be the actual name of the repo ("documentation-website")? Do we think we need to include the added text describing OpenSearch, or would the reader likely already know what OpenSearch is if they're reading this file?

File: README.md [hunk 29]
```diff
@@ -2,12 +2,19 @@
...
 
 
-## How you can help
+## Contributing
 
-Community contributions remain essential in keeping this documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in contributing, please see the [Contribution](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file.
+Community contributions remain essential in keeping the documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in submitting an issue or contributing content, see [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (natebower) [inline README.md:10]:
```suggestion
Community contributions remain essential to keeping the documentation comprehensive, useful, well organized, and up to date. If you are interested in submitting an issue or contributing content, see [CONTRIBUTING](CONTRIBUTING.md). 
```

File: README.md [hunk 30]
```diff
@@ -2,12 +2,19 @@
...
 
-## How you can help
+## Contributing
 
-Community contributions remain essential in keeping this documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in contributing, please see the [Contribution](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file.
+Community contributions remain essential in keeping the documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in submitting an issue or contributing content, see [CONTRIBUTING](CONTRIBUTING.md). 
+
+See these important resources when you are considering contributing to the documentation:  
+
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md) - The style guide covers the style standards to be observed when creating OpenSearch content.
```
Comment (natebower) [inline README.md:14]:
```suggestion
- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md) -- The style guide covers the style standards to be observed when creating OpenSearch Project content.
```

File: README.md [hunk 31]
```diff
@@ -2,12 +2,19 @@
...
+- [Terms](TERMS.md) - The terms list contains the key OpenSearch terms and tips on how and when to use them.
```
Comment (natebower) [inline README.md:15]:
```suggestion
- [OpenSearch terms](TERMS.md) -- The terms list contains key OpenSearch terms and tips on how and when to use them.  
```

File: README.md [hunk 32]
```diff
@@ -2,12 +2,19 @@
...
+## Contributing
 
-Community contributions remain essential in keeping this documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in contributing, please see the [Contribution](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file.
+Community contributions remain essential in keeping the documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in submitting an issue or contributing content, see [CONTRIBUTING](CONTRIBUTING.md). 
+
+See these important resources when you are considering contributing to the documentation:  
+
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md) - The style guide covers the style standards to be observed when creating OpenSearch content.
+- [Terms](TERMS.md) - The terms list contains the key OpenSearch terms and tips on how and when to use them.  
+- [API Style Guide](API_STYLE_GUIDE.md) - The API style guide provides the basic structure for creating OpenSearch API documentation.
```
Comment (natebower) [inline README.md:16]:
```suggestion
- [API Style Guide](API_STYLE_GUIDE.md) -- The API Style Guide provides the basic structure for creating OpenSearch API documentation.
```

File: README.md [hunk 33]
```diff
@@ -2,12 +2,19 @@
...
+- [Formatting Guide](FORMATTING_GUIDE.md) - The OpenSearch documentation uses a modified version of the [just-the-docs](https://github.com/pmarsceill/just-the-docs) Jekyll theme. Use the formatting guide for an overview of the commonly used formatted elements and to learn how to add pages to the website.
```
Comment (natebower) [inline README.md:17]:
```suggestion
- [Formatting Guide](FORMATTING_GUIDE.md) -- The OpenSearch documentation uses a modified version of the [just-the-docs](https://github.com/pmarsceill/just-the-docs) Jekyll theme. The Formatting Guide provides an overview of the commonly used formatting elements and how to add a page to the website.
```

File: README.md [hunk 34]
```diff
@@ -21,141 +28,19 @@ If you encounter problems or have questions when contributing to the documentati
...
-
 ## Code of conduct
 
 This project has adopted an [Open Source Code of Conduct](https://opensearch.org/codeofconduct.html).
 
 
 ## Security
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Do **not** create a public GitHub issue.
```
Comment (natebower) [inline README.md:38]:
```suggestion
If you discover a potential security issue in this project, we ask that you notify AWS/Amazon Security using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Do **not** create a public GitHub issue.
```

Comment (natebower) [inline README.md:38]: Re: "AWS/Amazon Security": Which one? Also, if he hasn't already, I would consider having Weston review this statement, especially given that we don't appear to provide a "why" in relation to the last sentence.

File: README.md [hunk 35]
```diff
@@ -2,12 +2,19 @@
...
+The documentation repository contains the documentation for OpenSearch, the highly scalable and extensible open-source software suite for search, analytics, observability, and other data-intensive applications. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
 
 
-## How you can help
+## Contributing
 
-Community contributions remain essential in keeping this documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in contributing, please see the [Contribution](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file.
+Community contributions remain essential in keeping the documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in submitting an issue or contributing content, see [CONTRIBUTING](CONTRIBUTING.md). 
+
+See these important resources when you are considering contributing to the documentation:
```
Comment (natebower) [inline README.md:12]:
```suggestion
The following resources provide important guidance regarding contributions to the documentation:  
```

**Review by natebower — COMMENTED** [review]
@kolchfa-aws Please see my comments and changes and let me know if you have any questions. Thanks!

File: CONTRIBUTING.md [hunk 36]
```diff
@@ -1,60 +1,144 @@
...
-* Check existing open and recently merged pull requests to ensure that someone else hasn't addressed the problem already.
-* If the change requires significant work, open an issue where we can first discuss your request.
+Before contributing content, make sure to read the following resources:
+- [README](README.md)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md)
+- [API Style Guide](API_STYLE_GUIDE.md)
+- [Formatting Guide](FORMATTING_GUIDE.md) 
 
-To create a pull request:
+NOTE: Make sure that any documentation you submit is your work or work that you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution.
```
[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply CONTRIBUTING.md:55]: This is taken from the OpenSearch main repo but I can have Weston review.

File: README.md (hunk 34 above)
Comment (kolchfa-aws) [reply README.md:38]: This was also taken from the main OpenSearch repo.

Comment (natebower) [reply README.md]: Let's just have Weston take a quick look to ensure he's OK with it.

File: CONTRIBUTING.md [hunk 37]
```diff
@@ -1,60 +1,144 @@
...
-See the [LICENSE](LICENSE) file for our project's licensing. 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.
+
+1. Navigate to your cloned repository.
+
+1. Install [Ruby](https://www.ruby-lang.org/en/) if you don't already have it. We recommend [RVM](https://rvm.io/), but you can use any method you prefer:
+
+   ```
+   curl -sSL https://get.rvm.io | bash -s stable
+   rvm install 2.6
```
Comment (hdhalter) [inline CONTRIBUTING.md:85]: There is an issue to update the Ruby version to 2.7: https://github.com/opensearch-project/documentation-website/issues/4498
```suggestion
   rvm install 2.7
```

**Pull request merged**